admin/services/date.service.js

```diff
@@ -35,6 +36,7 @@ const dateService = {
 
   checkAndFormat: function (date, format) {
     if (!(date instanceof Date || date instanceof moment)) {
+      winston.warn(`Date parameter is not a Date or Moment object: ${date}`)
       return ''
```

Comment:
i wonder if we should be throwing an error here?  or at least returning false 🤔

Reply:
Yes, I've considered that, but if we did then we wouldn't be able to directly use the output of dateService in the UI, we would have to catch errors and provide defaults.